### PR TITLE
Hotfix/us3555 find groups close to me

### PR DESCRIPTION
**Pull request by charlie-retzler**

### CODE REVIEW

File: Gateway/crds-angular/MinistryPlatform.config [hunk 1]
```diff
@@ -214,7 +214,7 @@
   <add key="ParticipantGoalLearnFromSomeone" value="7005"/>
   <add key="ParticipantGoalMakeFriends" value="7006"/>
   
-  <add key="MaxGroupSearchResults" value ="32"/>
+  <add key="MaxGroupSearchResults" value ="32000"/>
```
Comment (IngageMaddoxT) [inline Gateway/crds-angular/MinistryPlatform.config]: big change

File: crossroads.net/app/group_finder/directives/group_profile/group_profile.controller.js [hunk 2]
```diff
@@ -40,20 +40,26 @@
...
+        responses.location.city + ', ' +
+        responses.location.state + ', ' +
+        responses.location.zip;
+
       GoogleDistanceMatrixService.distanceFromAddress(hostAddress, [
         participantAddress
-      ]).then(function(result) {
-        $scope.getGroupDistance = function() {
-          if(result[0].distance) {
+      ]).then(function (result) {
```
Comment (IngageMaddoxT) [inline crossroads.net/app/group_finder/directives/group_profile/group_profile.controller.js]: linting on?  extra spaces here.

File: crossroads.net/app/group_finder/join/join_results.html [hunk 3]
```diff
@@ -1,4 +1,7 @@
-<preloader full-screen='true' ng-show="result.loading"> </preloader>
+<preloader full-screen='true' ng-show="result.loading"> Waiting for google... geez </preloader>
+<div class="text-center lead" ng-if="result.loading">
```
Comment (IngageMaddoxT) [inline crossroads.net/app/group_finder/join/join_results.html]: line #1 ng-show, line #2 ng-if.  why not match?

File: Gateway/crds-angular/MinistryPlatform.config (hunk 1 above)
Comment (charlie-retzler) [inline Gateway/crds-angular/MinistryPlatform.config]: yes, we meant to make this as many as available, but didn't want to remove code.

**Pull request merged**